src/main/client_context.cpp

```diff
@@ -580,5 +581,13 @@ void ClientContext::runQuery(std::string query) {
     }
     return;
 }
+
+processor::WarningContext& ClientContext::getWarningContext() {
```

Comment:
Our naming convension is that, if you return a reference(not a constant reference), we call it getXXXXUnsafe() so rename to getWarningContextUnsafe()